pkg/scheduler/grb_email_jobs.go

```diff
@@ -150,23 +162,33 @@ func sendAsyncPastDueNoQuorumEmailJobFunction(ctx context.Context, scheduledJob
 		return err
 	}
 
+	logger = logger.With(logfields.EmailType("PastDueNoQuorum"))
+
 	store, err := scheduledJob.store()
 	if err != nil {
-		logger.Error("error getting store from scheduler", zap.Error(err))
+		logger.Error(errGettingStore.Error(), zap.Error(err))
```

Comment:
These all look great! Thanks for implementing Sam!

Paradigm thought, don't have to implement this here. Right now, the error message isn't getting wrapped, so it's not actionable higher up in the call chain. I don't think that matters for the moment with the current code, but it could be a useful paradigm. What do you think about wrapping the error? There are better ways than the double %w, but it would still wrap the error. I did some stuff work on this for MINT with database errors for [duplicate violations etc](https://github.com/CMS-Enterprise/mint-app/blob/a15ad8088c8f761b37a00c73bec64519219c9a58/pkg/sqlutils/errors.go#L15).

```suggestion
        wrappedErr := fmt.Errorf("%w: %w", errGettingStore, err)
		logger.Error(errGettingStore.Error(), zap.Error(wrappedErr))
		return wrappedErr
```